src/inc/til/point.h

```diff
@@ -58,7 +58,7 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
 
         constexpr explicit operator bool() const noexcept
         {
-            return (x > 0) & (y > 0);
+            return x >= 0 && y >= 0;

```

Comment:
won't the change to `>=` break something?

Reply:
ah i see, your PR body says no. yay!

> nothing seemed to have used that operator in OpenConsole

Have we used it in _Terminal_?

Reply:
Ah, I forgot to respond: It's not used anywhere. (I tested it by making the function private and compiling everything (apart from tests).)